src/components/Button.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export default ({ type, value }) => (
+    <div className="form-group">
+        <input type={type || "submit"} value={value} className="btn btn-primary" />
```

Comment:
Components says its a button you can add it like.
`input type='button'`

Reply:
the button can be of type "button" or "submit"